src/agent/README.md

```diff
@@ -39,11 +39,21 @@ After that, we drafted the initial code here, and any contributions are welcome.
 ## Getting Started
 
 ### Build from Source
-The rust-agent need to be built with rust newer than 1.37, and static linked with `musl`.
+The rust-agent need to be built with static linked with musl:
```

Comment:
Please can you put backticks around "musl" as in the original version.

Reply:
Actually, you can leave this as "musl" now as I've just added that word to our dictionary: https://github.com/kata-containers/tests/pull/2937.

Reply:
The sentence does not parse for me. What about
```
The rust-agent needs to be be built static and linked with musl
```
(need -> needs and removed extra with)

In addition, I wonder why we insist on `musl`? Maybe we should use `should`? On Fedora, it is not that easy to get `musl` to work because of some incompatibility between Fedora headers and the C++ source code being compiled. So I build using `make LIBC=gnu` and that works quite well.

@jodh-intel, what do you think?

Reply:
I think @lifupan and @liubin will have the latest details on musl.

Reply:
May be should we add another section for using `gnu`?